### ISSUE TITLE
Console Display Issue - Custom Local Authenticator Not Visible in Application Signing Method Section

### ISSUE THREAD

**Issue by NilukaSripalim** [wso2/product-is#19001]
**Describe the issue:**
The custom local authenticator created is not displaying in the application signing method section of the console application. However, the authenticator is visible in the management console, and the authentication flow can be successfully completed through the management console. 
Additionally, after adding the authenticator via the management console, it does not appear in the console, application signing method section, showing as empty.

**Console view** 
<img width="777" alt="Screenshot 2024-01-18 at 15 48 17" src="https://github.com/wso2/product-is/assets/39120228/7d630b79-fe15-4f13-b3cb-77cdaef11098">

**Management Console**
<img width="1698" alt="Screenshot 2024-01-18 at 15 48 07" src="https://github.com/wso2/product-is/assets/39120228/78fd5d03-3b44-465b-b2aa-4773e6317926">

[customAuth.webm](https://github.com/wso2/product-is/assets/39120228/d25ff354-38c7-455b-a568-0bcf896c374c)

**How to reproduce:**
refer doc : https://is.docs.wso2.com/en/latest/references/extend/federation/write-a-custom-local-authenticator/

1. Create a custom local authenticator using the appropriate steps.
2. Confirm its visibility and functionality in the management console.
3. Check the application signing method section in the console application.

**Expected behavior:**
The custom local authenticator should be displayed in the application signing method section of the console application after creation. The list should remain consistent with what is visible in the management console.

**Environment information** (_Please complete the following information; remove  any unnecessary fields_) **:**
 - Product Version: wso2is-7.0.0-beta6-SNAPSHOT
 - OS: Mac
 - Database: H2
 - Userstore: JDBC

**Comment by ZiyamSanthosh** [comment]
**Analysis**

The authenticators are shown in the console UI only if the `isEnable` property is assigned as true. Adding such custom connectors through `/dropins` does not assign this property as true. Hence, the authenticator is not visible in the console UI.

<img width="592" alt="Screenshot 2024-01-19 at 10 59 11" src="https://github.com/wso2/product-is/assets/50801227/be7cc6dc-799c-4f62-b490-62fe0d71348b">

**Comment by ZiyamSanthosh** [comment]
When a local authenticator jar is added to the `/dropins` folder, it can be accessed and utilized through the management console. But, properties such as `isEnabled` should be explicitly mentioned through the configuration files. If not, such properties will be set to default which is false in this case. To view those authenticators in the connections page, the `isEnabled` attribute should be true. By adding a configuration as given below in the `deployment.toml` this can be achieved.
```
[[authentication.custom_authenticator]]
name = <authenticator name>
enabled = true
```
This has to be added as a step to our documentation since this is not explicitly mentioned.

**Comment by ZiyamSanthosh** [comment]
After adding the configuration as mentioned above, the authenticator is still not displayed in the Connections page. But the authenticator is seen in the sign-in method Tab under an application page. This is because, the authenticators shown in the Connections page are not decided according to the API response sent. This has to be validated and fixed from the frontend.
cc: @NipuniBhagya

**Comment by DilshanSenarath** [comment]
Since the custom local authenticator doesn't have any configuration fields, it is not displaying in `connections` section. Instead, any added custom local authenticator can be utilized directly in an application without the need for additional configuration. Following an offline discussion with Maduranga and Ziyam, it's clarified that there's no requirement to configure custom local authenticators. Hence, it's expected that custom local authenticators won't be displayed in the connections section. Hence, Closing the issue.

**Comment by NilukaSripalim** [comment]
> Since the custom local authenticator doesn't have any configuration fields, it is not displaying in `connections` section. Instead, any added custom local authenticator can be utilized directly in an application without the need for additional configuration. Following an offline discussion with Maduranga and Ziyam, it's clarified that there's no requirement to configure custom local authenticators. Hence, it's expected that custom local authenticators won't be displayed in the connections section. Hence, Closing the issue.

@DilshanSenarath Does this mean? We do not support custom local connectors through the new console.

**Comment by DilshanSenarath** [comment]
> @DilshanSenarath Does this mean? We do not support custom local connectors through the new console.

No, we can add custom local authenticators, but they are only visible in the application sign-in method tab, not under the connections.
